src/index.ts

```diff
@@ -74,11 +133,43 @@ app.get("/feed", async (req, res) => {
   // 5. order the posts by the field `updated_at` descending or ascending basesd on the parameter `orderBy`
   // 6. if the `searchString` parameter is not an empty, use the string to filter posts not matching the post titles or post content
 
-  const posts = await prisma.post.findMany();
-
+  const posts = await prisma.post.findMany({
+    where: {
+      AND: [
+        {
+          published: true,
+        },
+        {
+          OR: [
```

Comment:
if the `searchString` is empty we don't want to run this `OR`. Use a ternary to return an empty object in that case